reflex/utils/prerequisites.py

```diff
@@ -1925,12 +1925,11 @@ def format_address_width(address_width: str | None) -> int | None:
         return None
 
 
-@functools.lru_cache(maxsize=None)
-def get_cpu_info() -> CpuInfo | None:
-    """Get the CPU info of the underlining host.
+def _load_cpu_info() -> CpuInfo | None:
+    """Load the CPU info of the underlining host.
```

Comment:
seems like this should be _retrieve_cpu_info

```suggestion
def _load_cpu_info() -> CpuInfo | None:
    """Load the CPU info of the underlying host.
```